src/dscanner/analysis/mismatched_args.d

```diff
@@ -126,16 +127,16 @@ final class ArgVisitor : ASTVisitor
 	{
 		import dsymbol.string_interning : internString;
 
-		if (unary.primaryExpression is null)
-			return;
-		if (unary.primaryExpression.identifierOrTemplateInstance is null)
-			return;
-		if (unary.primaryExpression.identifierOrTemplateInstance.identifier == tok!"")
-			return;
-		immutable t = unary.primaryExpression.identifierOrTemplateInstance.identifier;
-		lines ~= t.line;
-		columns ~= t.column;
-		args ~= internString(t.text);
+		if (const IdentifierOrTemplateInstance iot = safeAccess(unary)
```

Comment:
Wouldn't `const iot` work too?

Reply:
Yes. It can be removed unless DCD completion is needed.

Reply:
Wrong, the Type is actually required, otherwise the internal wrapper is returned.

Reply:
with `unwrap()` inference works

Reply:
So why not use `unwrap` then?

Reply:
That's more o matter of style. I tend to use the type when i know it. Also better for DCD.